### PR TITLE
Make sure the period filter formatting supports both Chrome and FireFox

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4283

**Screen-cast**

Peek 2021-05-13 07-36

### CODE REVIEW

File: components/budget/filters/PeriodFilter.js [hunk 1]
```diff
@@ -72,7 +72,13 @@ const DateRange = ({ from, to }) => {
   if (!from && !to) {
     return <FormattedMessage id="DateRange.All" defaultMessage="All" />;
   } else if (from && to) {
-    return <FormattedDateTimeRange from={from} to={to} />;
+    return (
+      <FormattedMessage
+        id="Date.DateRange"
+        defaultMessage="{dateFrom, date, short}-{dateTo, date, short}"
```
Comment (kewitz) [inline components/budget/filters/PeriodFilter.js:78]: No strong opinion here, it looks great, but should we take the opportunity to make this a bit more verbose?
```suggestion
        defaultMessage="{dateFrom, date, short} to {dateTo, date, short}"
```
cc @Betree

**Review by kewitz — APPROVED** [review]

File: components/budget/filters/PeriodFilter.js (hunk 1 above)
Comment (SudharakaP) [reply components/budget/filters/PeriodFilter.js:78]: @kewitz : I agree too, this seems a bit more clearer. It's upto you @Betree; let me know. 😄

Comment (Betree) [reply components/budget/filters/PeriodFilter.js:78]: That's a good idea, and it's internationalized anyway so translators will be able to put whatever works best for their locale

**Review by Betree — APPROVED** [review]
Nice catch @SudharakaP :+1:

**Comment by SudharakaP** [comment]
> Nice catch @SudharakaP +1

Thanks much. 😄 😄

**Pull request merged**